data-gen/src/main/java/io/smartcat/data/loader/util/RandomizerImpl.java

```diff
@@ -0,0 +1,50 @@
+package io.smartcat.data.loader.util;
+
+import java.util.Random;
+
+/**
+ * Wrapper for java.util.Random.
+ */
+public class RandomizerImpl implements Randomizer {
```

Comment:
There is no need for interface for the wrapper. Move this implementation to Randomizer